pytests/repo/setup-repo.sh

```diff
@@ -80,13 +80,23 @@ repodir=${TEST_REPO_DIR}/yum.repos.d
 cachedir=${TEST_REPO_DIR}/cache/tdnf
 EOF
 
+SHA1SUM=$(sha1sum ${PUBLISH_PATH}/repodata/repomd.xml | cut -d' ' -f1)
+SHA256SUM=$(sha256sum ${PUBLISH_PATH}/repodata/repomd.xml | cut -d' ' -f1)
+MD5SUM=$(md5sum ${PUBLISH_PATH}/repodata/repomd.xml | cut -d' ' -f1)
```

Comment:
Add some more negative test cases for failures, like invalid digest name, invalid protocol, invalid md5sum and so on

Reply:
Yeah, will add some more test cases including negative test case for testing the failures.

Reply:
Added.